docs/dev_guides/core_concepts.md

```diff
@@ -0,0 +1,44 @@
+# Core concepts
+
+This package provides reusable components (*`threedee` objects*) which 
+- enable the use of certain custom 3D interactive functionality
+- simplify the development of custom 3D interactive functionality
+
+Broadly, *`threedee` objects* can be split into two categories, **manipulators** and **annotators**.
```

Comment:
I think it would be nice to add a paragraph about the ready-made plugins. I think this is a useful that that some people will want to use independently.

Reply:
nice! totally agree - adding now :)